### PR TITLE
Making parsing of eln_data.yaml and oasis.specific.yaml config file working plus examples

### PR DESCRIPTION
**Pull request by mkuehbach**

### CODE REVIEW

**Review by sanbrock — APPROVED** [review]
please note the few comments, I left

File: eln.batch.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# call from within the top-level directory of pynxtools_em
+dsrc="../../../../paper_paper_paper/scidat_nomad_ebsd/bb_analysis/data/development_spiecker/"
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply eln.batch.sh:4]: No, still needs to find a place on some server for the examples and sharing statements

File: pynxtools_em/concepts/nxs_concepts.py [hunk 2]
```diff
@@ -17,28 +17,27 @@
 #
 """Implement NeXus-specific groups and fields to document software and versions used."""
 
-# pylint: disable=no-member,too-few-members
+from pynxtools_em.shared.mapping_functors import variadic_path_to_specific_path
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply pynxtools_em/concepts/nxs_concepts.py:20]: Yeah it was there in the past havent heard anybody using it so I went to put it in the plugin but yeah that could be useful also for others potentially but right now we have the configuration of the plugins not fully harmonized

File: pynxtools_em/reader.py [hunk 3]
```diff
@@ -165,26 +171,32 @@ def read(
...
-            resolved_path = nxs_plt.nxs_mtex_get_nxpath_to_default_plot(
-                entry_id, file_paths[0]
-            )
-            # print(f"DEFAULT PLOT IS {resolved_path}")
-            if resolved_path != "":
-                nxs_plt.annotate_default_plot(template, resolved_path)
-
-        debugging = False
-        if debugging is True:
+        # run_block = False
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply pynxtools_em/reader.py:174]: Indeed a remnant of the past that will be removed when reactivating the default plot resolver

**Pull request merged**